foundations/javascript_basics/fundamentals-4.md

```diff
@@ -5,49 +5,47 @@
 
 ### Test Driven Development
 
-Test Driven Development \(TDD\) is a phrase you often hear in the dev world.  It refers to the practice of writing automated tests that describe how your code should work before you actually write the code.  For example, if you want to write a function that adds a couple of numbers, you would first write a test that uses the function and supplies the expected output.  The test will fail before you write your code, and you should be able to know that your code works correctly when the test passes.
+Test Driven Development \(TDD\) is a phrase you often hear in the dev world. It refers to the practice of writing automated tests that describe how your code should work before you actually write the code. For example, if you want to write a function that adds a couple of numbers, you would first write a test that uses the function and supplies the expected output. The test will fail before you write your code, and you should be able to know that your code works correctly when the test passes.
 
-In many ways, TDD is much more productive than writing code without tests.  If we didn't have the test for the adding function above, we would have to run the code ourselves over and over, plugging in different numbers until we were sure that it was working... not a big deal for a simple `add(2, 2)`, but imagine having to do that for more complicated functions, like checking whether or not someone has won a game of tic tac toe: \(`game_win(["o", null,"x",null,"x",null,"x", "o", "o"])`). If you didn't do TDD, then you might actually have to play multiple games against yourself just to test if the function was working correctly!
+In many ways, TDD is much more productive than writing code without tests. If we didn't have the test for the adding function above, we would have to run the code ourselves over and over, plugging in different numbers until we were sure that it was working... not a big deal for a simple `add(2, 2)`, but imagine having to do that for more complicated functions, like checking whether or not someone has won a game of tic tac toe: \(`game_win(["o", null,"x",null,"x",null,"x", "o", "o"])`). If you didn't do TDD, then you might actually have to play multiple games against yourself just to test if the function was working correctly!
 
-We will teach you the art of actually writing these tests later in the course.  The following exercises have the tests already written out for you. All you have to do is read the specs and write the code that makes them pass!  The very first exercise \(`01-helloWorld`\) is intentionally very simple and walks you through the process of running the tests and making them pass.  
+We will teach you the art of actually writing these tests later in the course. The following assignment has the tests already written out for you. All you have to do is set up the testing environment, read the specs, and write the code that makes them pass!
 
-### Good Luck!
+### Assignment
```

Comment:
I believe the term "Practice" goes well for this section since it has users do exercises which is essentially "practicing".
Can we change "Assignment" to "Practice" here?